code/game/objects/structures/bonfire.dm

```diff
@@ -0,0 +1,153 @@
+/////////BONFIRES//////////
```

Comment:
#176 - there aren't any conflicts, are there?

Reply:
this was just moved into it's folder